docs/source/tt-metalium/tt_metal/examples/matmul_multi_core_optimizations/data_reuse.rst

```diff
@@ -40,7 +40,7 @@ In addition to our double-buffer config, we introduce a third circular buffer de
 
 .. code-block:: cpp
 
-    uint32_t output_cb_index = CB::c_out0; // output operands start at index 16
+    uint32_t output_cb_index = CBIndex::c_16;
     uint32_t interm0_cb_index = 24; // Index for the intermediate circular buffer
```

Comment:
Please update all intermediate cb indexes as well.

Reply:
Should be done.